visualization/app/codeCharta/ui/resetStateButton/confirmResetStateDialog/confirmResetStateDialog.component.html

```diff
@@ -0,0 +1,8 @@
+<mat-toolbar class="header" xmlns="http://www.w3.org/1999/html">
+	<h2>Confirm reset map to default</h2>
```

Comment:
Would you please add a short explanation what is going to be reset?
E.g. "Uploaded Maps and settings are reset to defaults"